app/src/pages/attachment/index.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import Link from 'next/link';
+
+const Attachment = () => {
+
+    const uploadPhoto = async (e: React.ChangeEvent<HTMLInputElement>) => {
```

Comment:
Once you set up `S3ApiClient`, most of this logic will move into there so it's not in the UI :) it's good for maintainability to split up UI and business logic

Reply:
We can address this comment and set up a client in a separate CR, I'd rather merge and test your commit :)